pom.xml

```diff
@@ -444,10 +416,12 @@
             <groupId>com.buschmais.jqassistant.plugin</groupId>
             <artifactId>java</artifactId>
         </dependency>
+        <!-- todo java 11 -->
         <dependency>
```

Comment:
Following dependencies need to be removed:
RDBMS (moved to jqassistant-plugin)
M2-Repo (moved to jqassistant-plugin)
Spring (moved to jqassistant-plugin)
Asciidoc Report (moved to jqassistant-plugin)
GraphQL (moved to jqassistant-plugin)
GraphML (moved to jqassistant-plugin)
JavaEE 6, JPA2, EJB 3, CDI (moved to jqassistant-plugin)
OSGI (archived)
TestNG (archived)
JaxRS (archived)
Tycho (archived)
YAML 1 (archived)

Reply:
Ok, I will start to remove these dependencies.

Reply:
Removed.